examples/gallery/magnetogram_active_regions.py

```diff
@@ -0,0 +1,107 @@
+"""
+=========================================================
+Overplotting Active Region locations on magnetogram Plots
+=========================================================
+
+This example shows how to overplot Active Region location on magnetogram plots.
+"""
+
+##############################################################################
+# Start by importing the necessary modules.
+
+from sunpy.net import Fido, attrs as a
+from astropy import units as u
+import sunpy.map
+import matplotlib.pyplot as plt
+import numpy as np
+
+from sunpy.io.special import srs
+import sunpy.coordinates
+from astropy.coordinates import SkyCoord
+
+##############################################################################
+# Let's select a date (yyyy-mm-dd) for which we will be downloading files.
+
+day = "2017-01-01"
+
+##############################################################################
+# We will select a small time range to avoid downloading too many files.
+
+start_time = day + " 00:01:00"
+end_time = day + " 00:02:00"
```

Comment:
I'm not too happy to add strings starting with spaces. In this case I may suggest to put the date and time together as strings, eg:
```python
start_time = "2017-01-01 00:01:00"
end_time = "2017-01-01 00:02:00"
```
or to do it using `datetime` like
```python
start_time = datetime.datetim(2017, 1, 1, 0, 1, 0)
end_time = start_time + datetime.timedelta(0, 60)
```
For this example I probably would go with the first option, but I don't see the point to have strings with dates when python treats dates properly (it's just reminds me too much to idl)